doc/Installation/Installation-Ubuntu-1604-Nginx.md

```diff
@@ -142,9 +159,31 @@ Run validate.php as root in the librenms directory:
 
 ```bash
 cd /opt/librenms
+```
+
+```bash
 ./validate.php
 ```
+example output (versions may vary)
```

Comment:
Please remove this. Showing the output isn't necessary and the output of validate will be different depending on when people install.